paywall/src/stories/lock/Overlay-Optimistic.stories.js

```diff
@@ -154,7 +173,7 @@ const render = (
 storiesOf('Overlay/Optimistic Unlocking', module)
   .add('beginning purchase', () => {
     makeStore()
-    return render(locks, 'pending', undefined, undefined, {
+    return render(locks, 'pending', transaction, undefined, undefined, {
```

Comment:
nit, when you do this `undefined`, could you name the param so we know what this expects?
```suggestion
    return render(locks, 'pending', transaction, undefined /* errors */, undefined /* thisConfig*/, {
```

For example here, I was expecting to see `optimism` passed in too but that would be 6 args and we seem to expect 5?

Reply:
👍it does expect 6 args, the 6th is optimism. I implemented your suggestion throughout the file